docs/_templates/layout.html

```diff
@@ -25,11 +25,20 @@
 {% endblock %}
 
 {%- block content %}
-  <div id="settings" class="modal" style="display: none;" onclick="if (event.target == this){ closeModal(settings); }">
+  <div id="settings" class="modal" style="display: none;" onclick="if (event.target == this){ closeModal(settingsModal); }">
     <div class="modal-content">
-      <span class="close" onclick="closeModal(settings);" title="Close">&times;</span>
+      <span class="close" onclick="closeModal(settingsModal);" title="Close">&times;</span>
       <h1>Settings</h1>
-      <!-- TODO: ADD OPTIONS HERE -->
+
+      <div class='setting'>
+        <h3>Use a sans-serif font:
+          <label class="toggle" title="Use a sans serif font?">
```

Comment:
The `title` attribute here is somewhat redundant. More detail could help:

```suggestion
          <label class="toggle" title="Use a sans serif font? Your system font will be used, falling back to `sans-serif`.">
```